### DISCUSSION TITLE
Blazored Toast on Blazor Webassembly - How to set up service

### DISCUSSION THREAD

**Discussion by jbendler** [Blazored/Toast#187] [General]
Specifically, how do you set up the service on the client side of a webassembly blazor app?
I only have a Program.cs on the client side.  I am using .NET 6.

Update: I did it like this in my client's Program.cs:
	builder.Services.AddSingleton(typeof(IToastService), new ToastService());
	
	Can you add the correct code to your example that is on the blazored-toast page?

Thanks

**Comment by chrissainty** [comment]
@jbendler It shows you how to do this in the readme.

```csharp
builder.Services.AddBlazoredToast();
```
